gallery.html

```diff
@@ -6,6 +6,12 @@
     <title>Gallery</title>
 </head>
 <body>
-    
+    <h2 align="center">My Favorite Destination - My Home Town</h2>
```

Comment:
The 'align' attribute has been deprecated from the W3 standards.  Some browsers may still support this, but others may not.  Consider a different approach.